openedx/core/djangoapps/util/maintenance_banner.py

```diff
@@ -34,7 +34,7 @@ class MyView(View):
     """
     @wraps(func)
     def _decorated(request, *args, **kwargs):  # pylint: disable=missing-docstring
-        if waffle().is_enabled(DISPLAY_MAINTENANCE_WARNING):
+        if DISPLAY_MAINTENANCE_WARNING.is_enabled():
```

Comment:
Have no fear, nizar is here \o/ 😛 

Adding some tests for that 👍🏼 I'll let you know when they are ready (because I'll be testing them in the edX pipeline 😩). I know I should be testing them locally, but devstack is going to take time to provision 😞

Reply:
Alright, things should be ready now 😃